src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlAssertion.cs

```diff
@@ -152,6 +157,46 @@ public string MinorVersion
         /// </summary>
         public Signature Signature { get; set; }
 
+        /// <summary>
+        /// Gets the canonicalized (ExclusiveC14n) representation without comments.
```

Comment:
Perhaps we should document that null will be returned in a case that we can't get the canonical string (we swallow all exceptions).

Reply:
Perhaps we shouldn't return anything if the XmlTokenStream is not set.
We may want to make this internal for this release.
It is used in two places:
1. When performing OnBehalfOf
2. For Replaying of tokens, we want the same string.

Reply:
Made this internal.